src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1191,6 +1235,10 @@ because xbuild doesn't support framework reference assemblies.
 	Inputs="$(MSBuildProjectFullPath);$(MSBuildAllProjects);@(_AndroidResolvedResources);$(_AndroidBuildPropertiesCache)"
 	Outputs="$(_AndroidResFlagFile)"
 	DependsOnTargets="$(_OnResolveMonoAndroidSdks)">
+	<CheckForInvalidResourceFileNames
```

Comment:
This feels like a "project compatibility issue" waiting to drop: project builds *now*, in future we make `aapt2` the default, project *stops building*.

Ugh. :-(

Reply:
This is applying the same rules that `aapt` does and only when in `aapt2` mode.